src/main/java/net/wurstclient/commands/DistCmd.java

```diff
@@ -0,0 +1,94 @@
+package net.wurstclient.commands;
+
+import net.minecraft.util.math.BlockPos;
+import net.wurstclient.command.CmdException;
+import net.wurstclient.util.ChatUtils;
+import net.wurstclient.command.CmdError;
+import net.wurstclient.command.CmdException;
+import net.wurstclient.command.CmdSyntaxError;
+import net.wurstclient.command.Command;
+import java.lang.Math;
+
+
+public class DistCmd extends Command {
+
+
+    public DistCmd() {
+        super("dist", "Gets Distance between 2 Cordinates\nYou can convert Overworld Cords To Nether cords With cvt:<x> <z>\n ", ".dist <x> <z>:<x> <z>", ".dist cvt:<x> <z>");

```

Comment:
I don't think other commands with position elements use a colon delimiter. Instead why not call the args <x1> <z1> <x2> <z2>?